server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/Method.java

```diff
@@ -123,7 +127,15 @@ public String toString() {
     Request.Name requestHandled();
 
     Class<? extends JmapRequest> requestType();
-    
-    Stream<JmapResponse> process(JmapRequest request, MethodCallId methodCallId, MailboxSession mailboxSession);
 
+    default Flux<JmapResponse> process(JmapRequest request, MethodCallId methodCallId, MailboxSession mailboxSession) {
```

Comment:
We try to avoid Reactor in interfaces.

Reply:
That's internal within the JMAP module and not exposed outside it.

Not an issue here IMO